### ISSUE TITLE
The worker node can't join the cluster on ocp4.5.x

### ISSUE THREAD

**Issue by levinwang88** [RedHatOfficial/ocp4-helpernode#153]
I have tried to install the cluster with OCP (4.5.1,4.5.2,4.5.6,4.5.7) but the worker cannot join to cluster. 
Here is the error information with worker node
ignition[776]:GET result: Internel Server Error
ignition[776]:GET https://api-int.ocp4.test.net:22623/config/worker: attempt #206
ignition[776]:GET result: Internel Server Error
ignition[776]:GET https://api-int.ocp4.test.net:22623/config/worker: attempt #207
ignition[776]:GET result: Internel Server Error
ignition[776]:GET https://api-int.ocp4.test.net:22623/config/worker: attempt #208
.........

But the node of master is normal, you can see information below.
#### \#oc get nodes
NAME                 STATUS   ROLES    AGE     VERSION
mpoc1.ocp4.test.net   Ready    master   8m37s   v1.18.3+b74c5ed

## The information wih Curl command 
curl -k  https://api-int.ocp4.test.net:22623/config/worker -v
* About to connect() to api-int.ocp4.test.net port 22623 (#0)
*   Trying 9.128.99.103...
* Connected to api-int.ocp4.poc.net (9.128.99.103) port 22623 (#0)
* Initializing NSS with certpath: sql:/etc/pki/nssdb
* skipping SSL peer certificate verification
* SSL connection using TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
* Server certificate:
* 	subject: CN=api-int.ocp4.test.net
* 	start date: Sep 01 03:18:37 2020 GMT
* 	expire date: Aug 30 03:18:39 2030 GMT
* 	common name: api-int.ocp4.test.net
* 	issuer: CN=root-ca,OU=openshift
> GET /config/worker HTTP/1.1
> User-Agent: curl/7.29.0
> Host: api-int.ocp4.test.net:22623
> Accept: */*
> 
< HTTP/1.1 500 Internal Server Error
< Content-Length: 0
< Date: Tue, 01 Sep 2020 03:58:26 GMT
< 
* Connection #0 to host api-int.ocp4.test.net left intact

**Comment by levinwang88** [comment]
## Information returned when curl worker.
curl -k  https://api-int.ocp4.test.net:22623/config/worker -v
* About to connect() to api-int.ocp4.test.net port 22623 (#0)
*   Trying 9.128.99.103...
* Connected to api-int.ocp4.poc.net (9.128.99.103) port 22623 (#0)
* Initializing NSS with certpath: sql:/etc/pki/nssdb
* skipping SSL peer certificate verification
* SSL connection using TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
* Server certificate:
* 	subject: CN=api-int.ocp4.test.net
* 	start date: Sep 01 03:18:37 2020 GMT
* 	expire date: Aug 30 03:18:39 2030 GMT
* 	common name: api-int.ocp4.test.net
* 	issuer: CN=root-ca,OU=openshift
> GET /config/worker HTTP/1.1
> User-Agent: curl/7.29.0
> Host: api-int.ocp4.test.net:22623
> Accept: */*
> 
< HTTP/1.1 500 Internal Server Error
< Content-Length: 0
< Date: Tue, 01 Sep 2020 03:58:26 GMT
< 
* Connection #0 to host api-int.ocp4.test.net left intact

**Comment by christianh814** [comment]
Did you accept the certificate? Starting with 4.4+, you NEED to accept the CSR certificate for every new node that joins the cluster when you install using UPI.

**Comment by levinwang88** [comment]
###This is the current CSR status. How can I accept the Certificate
oc get csr
NAME        AGE   SIGNERNAME                                    REQUESTOR                                                                   CONDITION
csr-7kvfx   22m   kubernetes.io/kube-apiserver-client-kubelet   system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
csr-jc2wk   22m   kubernetes.io/kubelet-serving                 system:node:wpoc1.ocp4.poc.net                                              Approved,Issued
csr-kt4cl   37m   kubernetes.io/kube-apiserver-client-kubelet   system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
csr-xtq8v   36m   kubernetes.io/kubelet-serving                 system:node:mpoc1.ocp4.poc.net                                              Approved,Issued

**Comment by levinwang88** [comment]
oc get co
NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
authentication                                       Unknown     Unknown       True       33m
cloud-credential                                     True        False         False      41m
cluster-autoscaler                                                                        
config-operator                                                                           
console                                                                                   
csi-snapshot-controller                    4.5.2     False       True          False      18m
dns                                        4.5.2     True        False         False      29m
etcd                                       4.5.2     False       True          True       33m
image-registry                                                                            
ingress                                                                                   
insights                                                                                  
kube-apiserver                             4.5.2     False       True          True       33m
kube-controller-manager                    4.5.2     True        False         False      30m
kube-scheduler                             4.5.2     True        False         False      30m
kube-storage-version-migrator              4.5.2     True        False         False      18m
machine-api                                                                               
machine-approver                                                                          
machine-config                             4.5.2     True        False         False      31m
marketplace                                                                               
monitoring                                                                                
network                                    4.5.2     True        False         False      34m
node-tuning                                4.5.2     True        False         False      33m
openshift-apiserver                        4.5.2     False       False         False      33m
openshift-controller-manager                         False       True          False      33m
openshift-samples                                                                         
operator-lifecycle-manager                 4.5.2     True        False         False      31m
operator-lifecycle-manager-catalog         4.5.2     True        False         False      31m
operator-lifecycle-manager-packageserver             False       True          False      31m
service-ca                                 4.5.2     True        False         False      32m
storage

**Comment by levinwang88** [comment]
[core@bootstrap ~]$ journalctl -b -f -u release-image.service -u bootkube.service
-- Logs begin at Tue 2020-09-01 14:25:03 UTC. --
Sep 01 15:09:54 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "openshift-install-manifests.yaml" configmaps.v1./openshift-install-manifests -n openshift-config as it already exists
Sep 01 15:09:54 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-aggregator-client-signer.yaml" secrets.v1./aggregator-client-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:09:54 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-control-plane-client-signer.yaml" secrets.v1./kube-control-plane-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:09:55 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-csr-signer-signer.yaml" secrets.v1./csr-signer-signer -n openshift-kube-controller-manager-operator as it already exists
Sep 01 15:09:55 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-initial-kube-controller-manager-service-account-private-key.yaml" secrets.v1./initial-service-account-private-key -n openshift-config as it already exists
Sep 01 15:09:56 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-kube-apiserver-to-kubelet-signer.yaml" secrets.v1./kube-apiserver-to-kubelet-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:09:56 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-loadbalancer-serving-signer.yaml" secrets.v1./loadbalancer-serving-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:09:56 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-localhost-serving-signer.yaml" secrets.v1./localhost-serving-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:09:57 bootstrap.ocp4.poc.net bootkube.sh[13372]: Skipped "secret-service-network-serving-signer.yaml" secrets.v1./service-network-serving-signer -n openshift-kube-apiserver-operator as it already exists
Sep 01 15:10:16 bootstrap.ocp4.poc.net bootkube.sh[13372]: E0901 15:10:16.682713       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused

**Comment by levinwang88** [comment]
Normal   Pulled     39m                  kubelet, mpoc1.ocp4.poc.net  Container image "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:53c31eef65ce401bafa0a12b7d0faf75b3e2027a445a6252e350ae67fc37877a" already present on machine
  Normal   Created    39m                  kubelet, mpoc1.ocp4.poc.net  Created container kube-apiserver-insecure-readyz
  Warning  Unhealthy  38m                  kubelet, mpoc1.ocp4.poc.net  Readiness probe failed: Get https://9.111.143.244:6443/healthz: read tcp 9.111.143.244:52090->9.111.143.244:6443: read: connection reset by peer
  Warning  Unhealthy  38m                  kubelet, mpoc1.ocp4.poc.net  Readiness probe failed: Get https://9.111.143.244:6443/healthz: read tcp 9.111.143.244:52260->9.111.143.244:6443: read: connection reset by peer
  Normal   Pulled     38m (x3 over 39m)    kubelet, mpoc1.ocp4.poc.net  Container image "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a99ba49caf5063f24f139b2eb9326122364592c95ae72c7d0112d2cee1d0e30c" already present on machine
  Normal   Started    38m (x3 over 39m)    kubelet, mpoc1.ocp4.poc.net  Started container kube-apiserver
  Normal   Created    38m (x3 over 39m)    kubelet, mpoc1.ocp4.poc.net  Created container kube-apiserver
  Warning  Unhealthy  34m                  kubelet, mpoc1.ocp4.poc.net  Readiness probe failed: Get https://9.111.143.244:6443/healthz: read tcp 9.111.143.244:54736->9.111.143.244:6443: read: connection reset by peer
  Warning  BackOff    14m (x106 over 38m)  kubelet, mpoc1.ocp4.poc.net  Back-off restarting failed container
  Warning  Unhealthy  8m57s (x2 over 14m)  kubelet, mpoc1.ocp4.poc.net  (combined from similar events): Readiness probe failed: Get https://9.111.143.244:6443/healthz: read tcp 9.111.143.244:44802->9.111.143.244:6443: read: connection reset by peer

**Comment by levinwang88** [comment]
curl -k https://9.111.143.244:6443/healthz -v
* About to connect() to 9.111.143.244 port 6443 (#0)
*   Trying 9.111.143.244...
* Connected to 9.111.143.244 (9.111.143.244) port 6443 (#0)
* Initializing NSS with certpath: sql:/etc/pki/nssdb
* NSS error -5961 (PR_CONNECT_RESET_ERROR)
* TCP connection reset by peer
* Closing connection 0
curl: (35) TCP connection reset by peer

**Comment by levinwang88** [comment]
# curl -k  https://api-int.ocp4.poc.net:22623/config/worker -v
* About to connect() to api-int.ocp4.poc.net port 22623 (#0)
*   Trying 9.111.143.240...
* Connected to api-int.ocp4.poc.net (9.111.143.240) port 22623 (#0)
* Initializing NSS with certpath: sql:/etc/pki/nssdb
* skipping SSL peer certificate verification
* SSL connection using TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
* Server certificate:
* 	subject: CN=api-int.ocp4.poc.net
* 	start date: Sep 01 14:24:23 2020 GMT
* 	expire date: Aug 30 14:24:24 2030 GMT
* 	common name: api-int.ocp4.poc.net
* 	issuer: CN=root-ca,OU=openshift
> GET /config/worker HTTP/1.1
> User-Agent: curl/7.29.0
> Host: api-int.ocp4.poc.net:22623
> Accept: */*
> 
< HTTP/1.1 500 Internal Server Error
< Content-Length: 0
< Date: Tue, 01 Sep 2020 15:18:48 GMT
< 
* Connection #0 to host api-int.ocp4.poc.net left intact

**Comment by levinwang88** [comment]
##install-config.yaml file  ,I installed with proxy(restrict ip) and only had two nodes, one master and one Woker
apiVersion: v1
baseDomain: poc.net
compute:
- hyperthreading: Enabled
  name: worker
  replicas: 2
controlPlane:
  hyperthreading: Enabled
  name: master
  replicas: 1
metadata:
  name: ocp4
proxy:
  httpsProxy: http://9.123.99.34:1818
  httpProxy:  http://9.123.99.34:1818
  noProxy: localhost,127.0.0.1,.apps.ocp4.poc.net,.ocp4.poc.net,.apps.ocp4.poc.net,.ocp4.poc.net,.eng.platformlab.ibm.com,172.30.0.0/16,9.111.143.0/24,10.128.0.0/14
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  networkType: OpenShiftSDN
  serviceNetwork:
  - 172.30.0.0/16
platform:
  none: {}

**Comment by levinwang88** [comment]
#vars.yaml 
---
disk: "sda"
helper:
  name: "bastionocp45"
  ipaddr: "9.111.143.240"
  networkifacename: "ens192"
dns:
  domain: "poc.net"
  clusterid: "ocp4"
  forwarder1: "8.8.8.8"
  forwarder2: "8.8.4.4"
dhcp:
  router: "9.111.140.2"
  netmask: "255.255.255.0"
  bcast: "9.111.143.255"
  poolstart: "9.111.143.243"
  poolend: "9.111.143.245"
  ipid: "9.111.143.0"
  netmaskid: "255.255.255.0"
bootstrap:
  name: "bootstrap"
  ipaddr: "9.111.143.243"
  macaddr: "00:50:56:a1:26:aa"
masters:
  - name: "mpoc1"
    ipaddr: "9.111.143.244"
    macaddr: "00:50:56:a1:e4:3b"
workers:
  - name: "wpoc1"
    ipaddr: "9.111.143.245"
    macaddr: "00:50:56:a1:dc:2e"

**Comment by levinwang88** [comment]
#ls  /usr/local/src
dnsserial-DO_NOT_DELETE_BEFORE_ASKING_CHRISTIAN.txt  nfs-provisioner-rbac.yaml      openshift-install-linux.tar.gz
helm                                                 nfs-provisioner-sc.yaml        registry-pvc.yaml
nfs-provisioner-deployment.yaml                      openshift-client-linux.tar.gz

**Comment by levinwang88** [comment]
I do not know that one master and one worker if ok on ocp4.5 cluster. many thanks .

**Comment by levinwang88** [comment]
Sep 01 15:30:34 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:34.904108       1 reflector.go:251] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to watch *v1.Pod: Get https://localhost:6443/api/v1/pods?watch=true: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:35 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:35.905529       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:36 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:36.906875       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:37 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:37.908227       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:38 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:38.909210       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:39 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:39.910124       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:40 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:40.911201       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:41 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:41.912146       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:42 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:42.913117       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:43 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:43.914035       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:44 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:44.915162       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]

**Comment by levinwang88** [comment]
##kube-apiserver                             4.5.2     False       True          True       53m
kube-controller-manager                    4.5.2     True        False         False      49m
kube-scheduler                             4.5.2     True        False         False      50m
kube-storage-version-migrator              4.5.2     True        False         False      38m
machine-api                                                                               
machine-approver                                                                          
machine-config                             4.5.2     True        False         False      51m
marketplace                                                                               
monitoring                                                                                
network                                    4.5.2     True        False         False      54m
node-tuning                                4.5.2     True        False         False      53m
##openshift-apiserver                        4.5.2     False       False         False      53m

**Comment by levinwang88** [comment]
#oc get po  -n openshift-kube-apiserver
NAME                                   READY   STATUS             RESTARTS   AGE
installer-2-mpoc1.ocp4.poc.net         0/1     Completed          0          56m
installer-3-mpoc1.ocp4.poc.net         0/1     Completed          0          55m
kube-apiserver-mpoc1.ocp4.poc.net      3/4     CrashLoopBackOff   14         2m52s
revision-pruner-2-mpoc1.ocp4.poc.net   0/1     Completed          0          55m

**Comment by levinwang88** [comment]
#oc logs kube-apiserver-mpoc1.ocp4.poc.net -n openshift-kube-apiserver
9  <nil> 0 <nil>}. Err :connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused". Reconnecting...
W0901 15:34:04.644002       1 clientconn.go:1208] grpc: addrConn.createTransport failed to connect to {https://9.111.143.244:2379  <nil> 0 <nil>}. Err :connection error: desc = "transport: Error while dialing dial tcp 9.111.143.244:2379: connect: connection refused". Reconnecting...
W0901 15:34:05.232529       1 clientconn.go:1208] grpc: addrConn.createTransport failed to connect to {https://9.111.143.244:2379  <nil> 0 <nil>}. Err :connection error: desc = "transport: Error while dialing dial tcp 9.111.143.244:2379: connect: connection refused". Reconnecting...
W0901 15:34:05.246872       1 clientconn.go:1208] grpc: addrConn.createTransport failed to connect to {https://localhost:2379  <nil> 0 <nil>}. Err :connection error: desc = "transport: Error while dialing dial tcp [::1]:2379: connect: connection refused". Reconnecting...
Error: context deadline exceeded

**Comment by levinwang88** [comment]
Oddly enough, after restarting the worker, get nodes completely ready.
#oc get nodes
NAME                 STATUS   ROLES    AGE   VERSION
mpoc1.ocp4.poc.net   Ready    master   68m   v1.18.3+b74c5ed
wpoc1.ocp4.poc.net   Ready    worker   53m   v1.18.3+b74c5ed

**Comment by levinwang88** [comment]
### bootstrap is still in runing,
 #journalctl -b -f -u release-image.service -u bootkube.service
Last login: Tue Sep  1 15:28:15 2020 from 9.111.143.240
[core@bootstrap ~]$ journalctl -b -f -u release-image.service -u bootkube.service
-- Logs begin at Tue 2020-09-01 14:25:03 UTC. --
Sep 01 15:30:35 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:35.905529       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:36 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:36.906875       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:37 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:37.908227       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:38 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:38.909210       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:39 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:39.910124       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:40 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:40.911201       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:41 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:41.912146       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:42 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:42.913117       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:43 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:43.914035       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused
Sep 01 15:30:44 bootstrap.ocp4.poc.net bootkube.sh[17986]: E0901 15:30:44.915162       1 reflector.go:134] github.com/openshift/cluster-bootstrap/pkg/start/status.go:66: Failed to list *v1.Pod: Get https://localhost:6443/api/v1/pods: dial tcp [::1]:6443: connect: connection refused

**Comment by levinwang88** [comment]
I have fixed this problem with you comments. but  i found another issue . AVAILABLE is False of authentication&console NAME. here is the information.
NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
authentication                                       False       True          False      131m
cloud-credential                           4.5.6     True        False         False      168m
cluster-autoscaler                         4.5.6     True        False         False      155m
config-operator                            4.5.6     True        False         False      155m
console                                    4.5.6     False       True          True       131m

#oc get po -n openshift-console
NAME                         READY   STATUS             RESTARTS   AGE
console-669fbf45bc-6fxd8     0/1     Running            5          19m
console-669fbf45bc-rpf4v     0/1     CrashLoopBackOff   25         137m
downloads-6f9f7cdb56-2kz7t   1/1     Running            0          161m
downloads-6f9f7cdb56-r9sd5   1/1     Running            0          135m

**Comment by vsolasa** [comment]
@levinwang88 - I had a similar issue on 4.5.7 however my issue was on ppc64le architecture. I did two changes relevant to Power architecture - 1) modified the VCPU to 4VCPU and Ram to 16Gig 2) MTU setting should be 1450 and I modified it. Reinstalled the cluster with the above changes and my cluster came up without any issues.. i copied the below yaml file to manifests folder and created ignition files. I installed the cluster on KVM hosts though.. 

apiVersion: operator.openshift.io/v1
kind: Network
metadata:
  name: cluster
spec:
  defaultNetwork:
    type: OpenShiftSDN
    openshiftSDNConfig:
      mtu: 1400
